doc/src/manual/interacting-with-julia.md

```diff
@@ -324,4 +326,5 @@ messages in cyan you can add the following to your `juliarc.jl` file:
 ```julia
 ENV["JULIA_WARN_COLOR"] = :yellow
 ENV["JULIA_INFO_COLOR"] = :cyan
+ENV["JULIA_ERROR_COLOR"] = :magenta
```

Comment:
this undid https://github.com/JuliaLang/julia/pull/18628/commits/5a9b1d7950e2576f78d448599ecfb57dab97df5c